ocaml-lsp-server/src/custom_requests/custom_request.ml

```diff
@@ -5,3 +5,4 @@ module Switch_impl_intf = Req_switch_impl_intf
 module Typed_holes = Req_typed_holes
 module Type_enclosing = Req_type_enclosing
 module Wrapping_ast_node = Req_wrapping_ast_node
+module Documentation = Req_documentation
```

Comment:
Should be `Get_documentation` (and `req_documentation.ml(i)` should be `req_get_documentation`)